packages/grafana-data/src/utils/datasource.ts

```diff
@@ -16,7 +16,7 @@ import {
  * @public
  */
 export function getDataSourceRef(ds: DataSourceInstanceSettings): DataSourceRef {
-  return { uid: ds.uid, type: ds.type };
+  return { uid: ds.uid, type: ds.type, apiVersion: ds.meta?.apiVersion };
```

Comment:
Hymm -- do we we want to set this automaticly yet?  I see two big issues:
1. this assumes frontend and backend are in sync
2. This does not let the frontend say what apiVersion is is actually supporting